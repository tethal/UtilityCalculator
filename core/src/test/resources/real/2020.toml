[general]
start_date = 2020-01-01
end_date = 2020-12-31
tenant = ["Marie Černá", "Jindřišská 16", "111 50 Praha 1"]
owner = ["Jan Novák", "majitel@example.com"]
report_place = "V Praze"
report_date = 2021-05-02
sources = ["Vyúčtování služeb od Společenství vlastníků za rok 2020"]

[[deposits]]
description="celkem"
count=1
amount=40000

[[heating]]
annual_cost = 7727.3
start_date = 2020-01-01
end_date = 2020-12-31

[[other_fees]]
start_date = 2020-01-01
end_date = 2020-12-31
annual_cost = 9133

[[cold_water.reading]]
meter_id = "1"
reading_date = 2020-01-01
state = 0

[[cold_water.reading]]
meter_id = "1"
reading_date = 2021-01-01
state = 36.3


[[cold_water.tariff]]
start_date = 2020-01-01
end_date = 2020-12-31
unit_amount = 86.5161

[[hot_water.reading]]
meter_id = "1"
reading_date = 2020-01-01
state = 0

[[hot_water.reading]]
meter_id = "1"
reading_date = 2021-01-01
state = 25.9

[[hot_water.tariff]]
start_date = 2020-01-01
end_date = 2020-12-31
unit_amount = 86.343526

[[hot_water.heating_basic_cost]]
start_date = 2020-01-01
end_date = 2020-12-31
annual_cost = 1755

[[hot_water.heating_consumable_tariff]]
start_date = 2020-01-01
end_date = 2020-12-31
unit_amount = 128.698362
