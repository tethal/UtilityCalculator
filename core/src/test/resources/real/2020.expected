Report:
  dateRange: 2020-01-01...2021-01-01
  tenant: Marie Černá, Jindřišská 16, 111 50 Praha 1
  owner: Jan Novák, majitel@example.com
  placeDate: V Praze, 2021-05-02
  sources: Vyúčtování služeb od Společenství vlastníků za rok 2020
  sections:
    Studená voda: 3140.53
    Teplá voda: 7324.59
    Vytápění: 7727.30
    Ostatní poplatky: 9133.00
    Přijaté zálohy: -40000
    TOTAL: -12674.58

Studená voda:
  readings:
    [1] 2020-01-01...2021-01-01: 0...36.3 = 36.3
  priceList:
    2020-01-01...2021-01-01: 36.300 * 86.5161 = 3140.53

Teplá voda:
  readings:
    [1] 2020-01-01...2021-01-01: 0...25.9 = 25.9
  priceList:
    2020-01-01...2021-01-01: 25.900 * 86.343526 = 2236.30
  heatingBasicParts:
    2020-01-01...2021-01-01: 12.00 * 146.25 = 1755.00
  heatingConsumableParts:
    2020-01-01...2021-01-01: 25.900 * 128.698362 = 3333.29

Vytápění:
  2020-01: 1.00 * 7727.3 * 0.19 = 1468.19
  2020-02: 1.00 * 7727.3 * 0.16 = 1236.37
  2020-03: 1.00 * 7727.3 * 0.14 = 1081.82
  2020-04: 1.00 * 7727.3 * 0.09 = 695.46
  2020-05: 1.00 * 7727.3 * 0.02 = 154.55
  2020-06: 1.00 * 7727.3 * 0.0 = 0.00
  2020-07: 1.00 * 7727.3 * 0.0 = 0.00
  2020-08: 1.00 * 7727.3 * 0.0 = 0.00
  2020-09: 1.00 * 7727.3 * 0.01 = 77.27
  2020-10: 1.00 * 7727.3 * 0.08 = 618.18
  2020-11: 1.00 * 7727.3 * 0.14 = 1081.82
  2020-12: 1.00 * 7727.3 * 0.17 = 1313.64

Ostatní poplatky:
  2020-01-01...2021-01-01: 12.00 * 761.08 = 9133.00

Přijaté zálohy:
  celkem: 1 * 40000 = 40000
